source/_includes/common-tasks/commandline.md

```diff
@@ -102,6 +102,9 @@ Use "ha [command] --help" for more information about a command.
 {% if page.installation == "os" %}
 ### Console access
 
-You can also access the Home Assistant Operating System via a directly connected keyboard and monitor, the console. To log in to the physical console the username is `root`, with no password.
+You can also access the Home Assistant Operating System via a directly connected keyboard and monitor, the console. To log in to the physical console:
+
+1. Login as `root`, with no password.
+1. Type `login` and press enter to access the shell.
```

Comment:
But this is out of context here?

This page is about the CLI utility described above. That is what you have without entering the `login`?

Reply:
@frenck interesting! So you're saying maybe we should move the console access section under https://www.home-assistant.io/common-tasks/os/#configuring-access-to-files? Since this is kinda similar to the SSH access :thinking:

Reply:
I didn't say that. I said this addition on this page made makes no sense IMHO, as it's out of scope for the thing this page describes.